flame/models/parallelize_fla.py

```
@@ -153,18 +160,22 @@ def model_plan(self):
+                        use_local_output=not self.loss_parallel,
+                    ),
+                }
+            )
         else:
-            plans.update({
-                "lm_head": PrepareModuleWeight(layouts=Replicate()),
-                "criterion": LinearLossParallel()
-            })
+            plans.update(
+                {
+                    "lm_head": PrepareModuleWeight(layouts=Replicate()),
```

Ruff (0.8.2)

Undefined name `PrepareModuleWeight`

(F821)